### ISSUE TITLE
failing examples

### ISSUE THREAD

**Issue by jasmainak** [neuropycon/ephypype#17]
I get this error when trying to run an example on master:

```py
1.1.2
*** main_path -> /home/mainak/Desktop/projects/github_repos/BIDS-examples/sample_BIDS_omega ***
180906-14:52:55,261 nipype.workflow INFO:
	 Generated workflow graph: /home/mainak/Desktop/projects/github_repos/BIDS-examples/sample_BIDS_omega/preprocessing_pipeline/graph.png (graph2use=colored, simple_form=True).
180906-14:52:55,433 nipype.workflow INFO:
	 Workflow preprocessing_pipeline settings: ['check', 'execution', 'logging', 'monitoring']
180906-14:52:55,526 nipype.workflow INFO:
	 Running in parallel.
180906-14:52:55,572 nipype.workflow INFO:
	 [MultiProc] Running 0 tasks, and 1 jobs ready. Free memory (GB): 27.95/27.95, Free processors: 3/3.
180906-14:52:55,645 nipype.workflow INFO:
	 [Node] Setting-up "preprocessing_pipeline.datasource" in "/home/mainak/Desktop/projects/github_repos/BIDS-examples/sample_BIDS_omega/preprocessing_pipeline/_sess_index_ses-0001_subject_id_sub-0003/datasource".
180906-14:52:55,698 nipype.workflow INFO:
	 [Node] Running "datasource" ("nipype.interfaces.io.DataGrabber")
180906-14:52:55,725 nipype.workflow INFO:
	 [Node] Finished "preprocessing_pipeline.datasource".
180906-14:52:57,570 nipype.workflow INFO:
	 [Job 0] Completed (preprocessing_pipeline.datasource).
180906-14:52:57,573 nipype.workflow INFO:
	 [MultiProc] Running 0 tasks, and 1 jobs ready. Free memory (GB): 27.95/27.95, Free processors: 3/3.
180906-14:52:57,651 nipype.workflow INFO:
	 [Job 1] Cached (preprocessing_pipeline.preproc_meeg.ds2fif).
180906-14:52:59,672 nipype.workflow INFO:
	 [Job 2] Cached (preprocessing_pipeline.preproc_meeg.preproc).
180906-14:53:01,673 nipype.workflow INFO:
	 [Node] Setting-up "preprocessing_pipeline.preproc_meeg.ica" in "/home/mainak/Desktop/projects/github_repos/BIDS-examples/sample_BIDS_omega/preprocessing_pipeline/preproc_meeg/_sess_index_ses-0001_subject_id_sub-0003/ica".
180906-14:53:01,694 nipype.workflow INFO:
	 [Node] Running "ica" ("ephypype.interfaces.mne.preproc.CompIca")
Opening raw data file /home/mainak/Desktop/projects/github_repos/BIDS-examples/sample_BIDS_omega/preprocessing_pipeline/preproc_meeg/_sess_index_ses-0001_subject_id_sub-0003/preproc/sub-0003_task-rest_run-01_meg_raw_filt_dsamp.fif...
/home/mainak/Desktop/projects/github_repos/ephypype/ephypype/preproc.py:50: RuntimeWarning: This filename (/home/mainak/Desktop/projects/github_repos/BIDS-examples/sample_BIDS_omega/preprocessing_pipeline/preproc_meeg/_sess_index_ses-0001_subject_id_sub-0003/preproc/sub-0003_task-rest_run-01_meg_raw_filt_dsamp.fif) does not conform to MNE naming conventions. All raw files should end with raw.fif, raw_sss.fif, raw_tsss.fif, raw.fif.gz, raw_sss.fif.gz or raw_tsss.fif.gz
  raw = read_raw_fif(fif_file, preload=True)
This filename (/home/mainak/Desktop/projects/github_repos/BIDS-examples/sample_BIDS_omega/preprocessing_pipeline/preproc_meeg/_sess_index_ses-0001_subject_id_sub-0003/preproc/sub-0003_task-rest_run-01_meg_raw_filt_dsamp.fif) does not conform to MNE naming conventions. All raw files should end with raw.fif, raw_sss.fif, raw_tsss.fif, raw.fif.gz, raw_sss.fif.gz or raw_tsss.fif.gz
180906-14:53:03,573 nipype.workflow INFO:
	 [MultiProc] Running 1 tasks, and 0 jobs ready. Free memory (GB): 27.75/27.95, Free processors: 2/3.                     
                     Currently running:
                       * preprocessing_pipeline.preproc_meeg.ica
    Read 5 compensation matrices
    Range : 0 ... 479999 =      0.000 ...   599.999 secs
Ready.
Current compensation grade : 3
Reading 0 ... 479999  =      0.000 ...   599.999 secs...
Fitting ICA to data using 270 channels (please be patient, this may take a while)
Inferring max_pca_components from picks
    Rejecting  epoch based on MAG : [u'MLO12-4408', u'MLO13-4408', u'MLO23-4408']
Artifact detected in [294400, 296000]
Selection by explained variance: 27 components
Fitting ICA took 52.8s.
/home/mainak/Desktop/projects/github_repos/ephypype/ephypype/preproc.py:70: RuntimeWarning: This filename (/home/mainak/Desktop/projects/github_repos/BIDS-examples/sample_BIDS_omega/preprocessing_pipeline/preproc_meeg/_sess_index_ses-0001_subject_id_sub-0003/ica/sub-0003_task-rest_run-01_meg_raw_filt_dsamp_ica-tseries.fif) does not conform to MNE naming conventions. All raw files should end with raw.fif, raw_sss.fif, raw_tsss.fif, raw.fif.gz, raw_sss.fif.gz or raw_tsss.fif.gz
  ica_src.save(ica_ts_file)
This filename (/home/mainak/Desktop/projects/github_repos/BIDS-examples/sample_BIDS_omega/preprocessing_pipeline/preproc_meeg/_sess_index_ses-0001_subject_id_sub-0003/ica/sub-0003_task-rest_run-01_meg_raw_filt_dsamp_ica-tseries.fif) does not conform to MNE naming conventions. All raw files should end with raw.fif, raw_sss.fif, raw_tsss.fif, raw.fif.gz, raw_sss.fif.gz or raw_tsss.fif.gz
Writing /home/mainak/Desktop/projects/github_repos/BIDS-examples/sample_BIDS_omega/preprocessing_pipeline/preproc_meeg/_sess_index_ses-0001_subject_id_sub-0003/ica/sub-0003_task-rest_run-01_meg_raw_filt_dsamp_ica-tseries.fif
Closing /home/mainak/Desktop/projects/github_repos/BIDS-examples/sample_BIDS_omega/preprocessing_pipeline/preproc_meeg/_sess_index_ses-0001_subject_id_sub-0003/ica/sub-0003_task-rest_run-01_meg_raw_filt_dsamp_ica-tseries.fif [done]
Using channel ECG to identify heart beats.
Setting up band-pass filter from 8 - 16 Hz
Filter length of 8192 samples (10.240 sec) selected
Number of ECG events detected : 529 (average pulse 52 / min.)
529 matching events found
No baseline correction applied
Not setting metadata
180906-14:54:09,662 nipype.workflow WARNING:
	 [Node] Error on "preprocessing_pipeline.preproc_meeg.ica" (/home/mainak/Desktop/projects/github_repos/BIDS-examples/sample_BIDS_omega/preprocessing_pipeline/preproc_meeg/_sess_index_ses-0001_subject_id_sub-0003/ica)
180906-14:54:11,663 nipype.workflow ERROR:
	 Node ica.a0 failed to run on host mainak-ThinkPad-W540.
180906-14:54:11,663 nipype.workflow ERROR:
	 Saving crash info to /home/mainak/Desktop/crash-20180906-145411-mainak-ica.a0-77c8dd8b-5461-413e-ad45-fbe44a7adf75.pklz
Traceback (most recent call last):
  File "/home/mainak/anaconda2/lib/python2.7/site-packages/nipype-1.1.2-py2.7.egg/nipype/pipeline/plugins/multiproc.py", line 69, in run_node
    result['result'] = node.run(updatehash=updatehash)
  File "/home/mainak/anaconda2/lib/python2.7/site-packages/nipype-1.1.2-py2.7.egg/nipype/pipeline/engine/nodes.py", line 471, in run
    result = self._run_interface(execute=True)
  File "/home/mainak/anaconda2/lib/python2.7/site-packages/nipype-1.1.2-py2.7.egg/nipype/pipeline/engine/nodes.py", line 555, in _run_interface
    return self._run_command(execute)
  File "/home/mainak/anaconda2/lib/python2.7/site-packages/nipype-1.1.2-py2.7.egg/nipype/pipeline/engine/nodes.py", line 635, in _run_command
    result = self._interface.run(cwd=outdir)
  File "/home/mainak/anaconda2/lib/python2.7/site-packages/nipype-1.1.2-py2.7.egg/nipype/interfaces/base/core.py", line 521, in run
    runtime = self._run_interface(runtime)
  File "/home/mainak/Desktop/projects/github_repos/ephypype/ephypype/interfaces/mne/preproc.py", line 63, in _run_interface
    eog_ch_name, n_components, reject)
  File "/home/mainak/Desktop/projects/github_repos/ephypype/ephypype/preproc.py", line 89, in compute_ica
    ch_name=ecg_ch_name)
  File "<string>", line 2, in create_ecg_epochs
  File "/home/mainak/Desktop/projects/github_repos/mne-python/mne/utils.py", line 724, in verbose
    return function(*args, **kwargs)
  File "/home/mainak/Desktop/projects/github_repos/mne-python/mne/preprocessing/ecg.py", line 336, in create_ecg_epochs
    verbose=verbose, preload=preload)
  File "<string>", line 2, in __init__
  File "/home/mainak/Desktop/projects/github_repos/mne-python/mne/utils.py", line 724, in verbose
    return function(*args, **kwargs)
  File "/home/mainak/Desktop/projects/github_repos/mne-python/mne/epochs.py", line 2035, in __init__
    verbose=verbose)
  File "/home/mainak/Desktop/projects/github_repos/mne-python/mne/epochs.py", line 348, in __init__
    self.info = pick_info(self.info, picks)
  File "/home/mainak/Desktop/projects/github_repos/mne-python/mne/io/pick.py", line 408, in pick_info
    'channels' % (current_comp, comps_missing))
RuntimeError: Compensation grade 3 has been applied, but compensation channels are missing: [u'BG1-4408', u'BG2-4408', u'BG3-4408', u'G11-4408', u'G12-4408', u'G13-4408', u'G22-4408', u'G23-4408', u'P11-4408', u'P12-4408', u'P13-4408', u'P23-4408', u'Q11-4408', u'Q12-4408', u'Q23-4408', u'R12-4408', u'R13-4408', u'R22-4408']
Either remove compensation or pick compensation channels

180906-14:54:11,669 nipype.workflow INFO:
	 [MultiProc] Running 0 tasks, and 0 jobs ready. Free memory (GB): 27.95/27.95, Free processors: 3/3.
180906-14:54:13,642 nipype.workflow INFO:
	 ***********************************
180906-14:54:13,643 nipype.workflow ERROR:
	 could not run node: preprocessing_pipeline.preproc_meeg.ica.a0
180906-14:54:13,643 nipype.workflow INFO:
	 crashfile: /home/mainak/Desktop/crash-20180906-145411-mainak-ica.a0-77c8dd8b-5461-413e-ad45-fbe44a7adf75.pklz
180906-14:54:13,643 nipype.workflow INFO:
	 ***********************************
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-1-69d92eebc094> in <module>()
     88 
     89     # Run workflow locally on 3 CPUs
---> 90     main_workflow.run(plugin='MultiProc', plugin_args={'n_procs': 3})

/home/mainak/anaconda2/lib/python2.7/site-packages/nipype-1.1.2-py2.7.egg/nipype/pipeline/engine/workflows.pyc in run(self, plugin, plugin_args, updatehash)
    593         if str2bool(self.config['execution']['create_report']):
    594             self._write_report_info(self.base_dir, self.name, execgraph)
--> 595         runner.run(execgraph, updatehash=updatehash, config=self.config)
    596         datestr = datetime.utcnow().strftime('%Y%m%dT%H%M%S')
    597         if str2bool(self.config['execution']['write_provenance']):

/home/mainak/anaconda2/lib/python2.7/site-packages/nipype-1.1.2-py2.7.egg/nipype/pipeline/plugins/base.pyc in run(self, graph, config, updatehash)
    190 
    191         self._remove_node_dirs()
--> 192         report_nodes_not_run(notrun)
    193 
    194         # close any open resources

/home/mainak/anaconda2/lib/python2.7/site-packages/nipype-1.1.2-py2.7.egg/nipype/pipeline/plugins/tools.pyc in report_nodes_not_run(notrun)
     80                 logger.debug(subnode._id)
     81         logger.info("***********************************")
---> 82         raise RuntimeError(('Workflow did not execute cleanly. '
     83                             'Check log for details'))
     84 
```

**Comment by annapasca** [comment]
Hi @jasmainak it would be better to test the pipelines after we merge my dev branch to master one, since my last tests were on my dev branch

**Comment by jasmainak** [comment]
I understand that. Do you want me to try out your `dev` branch in that case?

**Comment by jasmainak** [comment]
@annapasca examples don't work even after merging `dev` branch

**Comment by annapasca** [comment]
Hi @jasmainak I looked at this crash and it seems related to new way of MNE to manage the compensation channels. I fixed it and I'll do a new PR after we solve the other problem I explained below. 

Indeed, there is a new crash at line 438 of preproc.py

`fig = ica_src.plot_psd(tmax=60, picks=[i_ic], fmax=140, show=False)`

It seems plot_psd works in a different way, it says 
`('No data channels found')`

Do u have some idea what changed in this MNE function?

**Comment by jasmainak** [comment]
hmm ... not the top of my head. Sorry !

**Comment by annapasca** [comment]
@dmalt do you have time to check this error? it is related to the plot of PSD of the ICA components
If you cannot could we put a try?

**Comment by dmalt** [comment]
@annapasca, I'm trying to download the OMEGA data but all the links from the [docs](https://neuropycon.github.io/neuropycon_doc/examples/examples.html) to the OMEGA webpages seem to be broken. Where can I get the data to test?
@jasmainak which example were you running exactly?

**Comment by annapasca** [comment]
Hi @dmalt ! I'm creating a PR about the bugs @jasmainak found.
There is a problem about the plot of the PSD of ICA component since the function plot_psd is no more used to this aim

**Comment by annapasca** [comment]
about OMEGA I'm putting to Dropbox 1 sbj so we can test it

**Comment by dmalt** [comment]
@annapasca, ok!

**Comment by annapasca** [comment]
@dmalt I created the PR so you can see the change

about Dropbox I'm waiting to have more space to upload the sample dataset. In the middle time I updated the link in the tutorial webpage :)

**Comment by jasmainak** [comment]
All good now
